flutter-app/lib/label_samples_grid.dart

```diff
@@ -65,19 +65,51 @@ class ListLabelSamples extends StatelessWidget {
           preferredSize: new AppBar().preferredSize,
         ),
         floatingActionButton: model.isLoggedIn()
-            ? new FloatingActionButton.extended(
-                icon: const Icon(Icons.add_a_photo),
-                label: const Text('Add Images'),
-                onPressed: () async {
-                  final c = Camera(dataset, labelName, labelKey, model);
-                  await c.init();
-                  Navigator.push(
-                    context,
-                    MaterialPageRoute(
-                      builder: (context) => c,
+            ? Align(
+                alignment: Alignment.bottomRight,
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  mainAxisSize: MainAxisSize.min,
+                  children: <Widget>[
+                    new FloatingActionButton.extended(
+                      icon: const Icon(Icons.add_a_photo),
+                      heroTag: 1,
+                      label: const Text('Add Images'),
```

Comment:
Having two buttons doesn't look that great :( Can we use a FAB menu or just use the icon?